src/network/networkd-gperf.gperf

```diff
@@ -30,8 +30,10 @@ Network.RouteTable,                      config_parse_route_table_names,
 Network.IPv4Forwarding,                  config_parse_tristate,                  0,          offsetof(Manager, ip_forwarding[0])
 Network.IPv6Forwarding,                  config_parse_tristate,                  0,          offsetof(Manager, ip_forwarding[1])
 Network.IPv6PrivacyExtensions,           config_parse_ipv6_privacy_extensions,   0,          offsetof(Manager, ipv6_privacy_extensions)
+DHCPv4.DefaultUseDomains,                config_parse_default_dhcp_use_domains,  0,          offsetof(Manager, dhcp_use_domains)
```

Comment:
Please use `UseDomains`, that is, please drop `Default`.

Reply:
Is there a reason why having "Default" in the naming is not suggested? Using a different name can better separate this parameter from the existing "UseDomains" field.

Reply:
Why the new setting needs to be separated from `UseDomains` in .network file?
Yes, for PID1, we have bunch of DefaultXYZ setting. But that's quite redundant I think, and other existing settings, e.g. `IPv4Forwarding=` or so does not have `Default` prefix.